### PR TITLE
Colors

### PR DESCRIPTION
**Pull request by MattAWright**

### CODE REVIEW

File: cmd/list.go [hunk 1]
```diff
@@ -5,16 +5,18 @@ import (
 	"fmt"
 	"os"
 	"sync"
-	"text/tabwriter"
+	//"text/tabwriter"
```
Comment (ianlivingstone) [inline cmd/list.go]: forgotten?

File: cmd/envs.go [hunk 2]
```diff
@@ -163,69 +153,92 @@ func listEnvsCmd(ctx *cli.Context) error {
...
-	}
-	if org == nil {
-		return errs.NewExitError("Org not found.")
+	var orgs []envelope.Org
+
+	// Retrieve the org name supplied via the --org flag.
+	// This flag is optional. If none was supplied, then
+	// orgFlagArgument will be set to "". In this case,
+	// prompt the user to select an org.
+	orgName := ctx.String("org")
```
Comment (ianlivingstone) [inline cmd/envs.go]: Could we dry this up? We repeat it in several different places.

File: cmd/invites.go [hunk 3]
```diff
@@ -25,9 +25,9 @@ func init() {
 				Name:  "list",
 				Usage: "List outstanding invitations for an organization. These invites have yet to be approved.",
 				Flags: []cli.Flag{
-					orgFlag("org to list invites for", true),
+					orgFlag("org to list invites for", false),
 					cli.BoolFlag{
-						Name:  "approved",
+						Name:  "all, a",
 						Usage: "Show only approved invites",
```
Comment (ianlivingstone) [inline cmd/invites.go]: This usage isn't correct anymore, is it? If so can you update it? We didn't actually update the behaviour either, it shows only approved invites. 

So, if it's going to be `--all` it should show all invites, if not, it should probably continue being `--approved`, the shortcut `-a` is 👍  either way!

File: cmd/invites_list.go [hunk 4]
```diff
@@ -4,16 +4,19 @@ import (
 	"context"
 	"fmt"
 	"os"
-	"text/tabwriter"
+	//"text/tabwriter"
```
Comment (ianlivingstone) [inline cmd/invites_list.go]: Forgotten?

File: cmd/orgs.go [hunk 5]
```diff
@@ -7,9 +7,10 @@ import (
 	"sort"
 	"strconv"
 	"sync"
-	"text/tabwriter"
+	//"text/tabwriter"
```
Comment (ianlivingstone) [inline cmd/orgs.go]: forgotten?

File: cmd/projects.go [hunk 6]
```diff
@@ -53,22 +53,67 @@ func init() {
...
-	fmt.Println(strings.Repeat("-", utf8.RuneCountInString(title)))
+	fmt.Printf("  %s\n", ui.Bold("Projects"))
 	for _, project := range projects {
-		fmt.Println(project.Body.Name)
+		fmt.Printf("  %s\n", project.Body.Name)
 	}
 	fmt.Println("")
 
+	count := strconv.Itoa(len(projects))
+	countStr := "Org " + orgName + " has (" + count + ") projects."
```
Comment (ianlivingstone) [inline cmd/projects.go]: You could do this with `fmt.Printf("Org "%s has (%d) projects.\n", orgName, len(projects))`

File: ui/color.go [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+func Color(c ansiterm.Color, s string) string {
+  ctx := ansiterm.Context {
+    Foreground: c,
+  }
+
+  return createStyledString(ctx, s)
+}
+
+func Environment(s string) string {
```
Comment (ianlivingstone) [inline ui/color.go]: Are we using these anywhere? if not, we should cut them loose.

File: errs/errs.go [hunk 8]
```diff
@@ -37,3 +37,14 @@ func NewExitError(message string) error {
 	}
 	return cli.NewExitError(message, -1)
 }
+
+func FilterErrors(errors ...error) cli.MultiError {
```
Comment (ianlivingstone) [inline errs/errs.go]: Instead of `FilterErrors` perhaps this should just be `MultiError` as its a MultiError wrapper?

File: cmd/invites.go (hunk 3 above)
Comment (MattAWright) [reply cmd/invites.go]: Yes, this was clumsy, misunderstood the behaviour. Have reverted it bad to "approved" but added the short "a". After taking another look, being able to show all approved vs all other types makes sense.

File: cmd/envs.go (hunk 2 above)
Comment (MattAWright) [reply cmd/envs.go]: Done

File: cmd/invites_list.go (hunk 4 above)
Comment (MattAWright) [reply cmd/invites_list.go]: Yep, fixed

File: cmd/list.go (hunk 1 above)
Comment (MattAWright) [reply cmd/list.go]: Yep

File: cmd/projects.go (hunk 6 above)
Comment (MattAWright) [reply cmd/projects.go]: Yeah now that you say this, it's weird I would ever do otherwise. This is everywhere so I'll change it in other places too.

**Pull request merged**